internal/webserver/endpoints.go

```diff
@@ -36,18 +34,9 @@ func (s *WebServer) SpotifyCallback(c echo.Context) error {
 	platform := platformCookie.Value
 	userID := userCookie.Value
 	authCode := c.QueryParam("code")
+	s.spotify.AddUser(authCode, platform, userID) //TODO get error
 
-	ctx := context.Background() //FIXME
-	token, err := s.authConfig.Exchange(ctx, authCode)
-	if err != nil {
-		s.server.Logger.Fatal(err)
-	}
-
-	client := s.authConfig.Client(ctx, token)
-	client.Get("...") //FIXME
-	//TODO strore to DB
-
-	return c.String(http.StatusOK, platform+"\n"+userID+"\n"+token.AccessToken)
+	return c.String(200, "Authentication Successful")
```

Comment:
I have a suggestion here: we can redirect to the bot with a deep link and a parameter I know this is possible with telegram bots.

Reply:
Yeah, good idea. and we can use an inline keyboard with a [switch_inline_query]{https://core.telegram.org/bots/api#inlinekeyboardmarkup} button to send the user back to the original chat.